cdist/conf/type/__install_config/files/remote/exec

```diff
@@ -36,14 +36,17 @@ shift
 
 # escape ' with '"'"'
 code="$(echo "$@" | sed -e "s/'/'\"'\"'/g")"
+# shellcheck disable=SC2089
 code="chroot $chroot sh -e -c '$code'"
 
 log "target_host: $target_host"
 log "chroot: $chroot"
-log "@: $@"
+log "@: $*"
 log "code: $code"
 
 # Run the code
-$__default_remote_exec $target_host $code
+# code should be split on spaces
+# shellcheck disable=SC2086,SC2090
+"$__default_remote_exec" "$target_host" $code
```

Comment:
Shouldn't the quotes around `$__default_remote_exec` be removed -- to allow splitting?

Reply:
You are right, they should.